istioctl/cmd/authz.go

```diff
@@ -45,30 +45,18 @@ var (
 var (
 	checkCmd = &cobra.Command{
 		Use:   "check <pod-name>[.<pod-namespace>]",
-		Short: "Check the TLS/JWT/RBAC settings based on Envoy config",
-		Long: `Check analyzes the TLS/JWT/RBAC settings directly based on the Envoy config. The Envoy config could
+		Short: "Check the authorization configuration based on Envoy config",
```

Comment:
Can it be a little bit more descriptive? What does it mean by `Check the authorization configuration based on Envoy config`? What does it do when it "checks" the envoy config, what does it check, etc. Similar to other places for this `check` command.

Reply:
Updated.